okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -468,8 +477,8 @@ public OkHttpChannelBuilder useTransportSecurity() {
    */
   public OkHttpChannelBuilder scheduledExecutorService(
       ScheduledExecutorService scheduledExecutorService) {
-    this.scheduledExecutorService =
-        checkNotNull(scheduledExecutorService, "scheduledExecutorService");
+    this.scheduledExecutorServicePool =
+        new FixedObjectPool<>(checkNotNull(scheduledExecutorService, "scheduledExecutorService"));
```

Comment:
I take it that "It's an optional parameter" in the JavaDoc is referring to scheduledExecutorService. It's not optional so the JavaDoc should change?

Reply:
I think it is meaning, "you don't have to call this method on the builder."

There is oddity here. Compare how `scheduledExecutorService()` handles null vs `transportExecutor()`. But that was existing, so I didn't touch it.